File: P34_1609/P34.c
#include<stdio.h>
int main()
{
    int n, m, s;
    printf("Enter number of students:");
    scanf("%d", &n);

    printf("Enter number of sweets:");
    scanf("%d", &m);

    printf("Enter chair number:");
    scanf("%d", &s);

    int ans = s + m%n - 1;
    int final_ans = ans % n;

    if(ans!=0)
        {
            printf("%d", final_ans);
        }
    else
        {
            printf("%d", n);
        }

    return 0;

}

File: P34_1609/P34.py
print("Enter number of students:")
n=int(input())
print("Enter number of sweets:")
m=int(input())
print("Enter chair number:")
s=int(input())
ans=s+m%n-1
final_ans=ans%n

if ans!=0:
  print(final_ans)
else:
  print(n)